.github/SECURITY.md

```diff
@@ -0,0 +1,10 @@
+Security Policy
+===============
+
+If you found any issues that might have security implications,
+please send a report to security[at]symfony.com
+DO NOT PUBLISH SECURITY REPORTS PUBLICLY.
+
+Our [Security Policy][1] is described in our official documentation.
```

Comment:
maybe remove the 2nd "our" by "the"

Reply:
Or remove it entirely:
The full [Security Policy][1] is described in the official documentation.

Reply:
√